library/src/amd_detail/rocblaslt/src/tensile_host.cpp

```diff
@@ -537,7 +537,7 @@ namespace
                 : "",
             problem.tensor(TensileLite::ContractionProblemGemm::TENSOR::E).strides().size()
                 ? std::to_string(
-                    problem.tensor(TensileLite::ContractionProblemGemm::TENSOR::E).strides()[1])
+                      problem.tensor(TensileLite::ContractionProblemGemm::TENSOR::E).strides()[1])
```

Comment:
Is this trailing whitespace?

Reply:
@mahmoodw Was this leading whitespace added automatically by the githook formatter?

Reply:
@bethune-bryant that's right